documentation/release_notes.rst

```diff
@@ -8,6 +8,58 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.3.0
+===============
+
+New Features
+------------
+- Added dynamic selection as an experimental feature. Provides functions such as ``select``,
+  ``submit`` and ``submit_and_wait``. It includes selection policies: ``fixed_resource_policy``,
+  ``round_robin_policy``, ``dynamic_load_policy``, ``auto_tune_policy``, and provides default
+  support for SYCL queues. 
+- ``unseq`` and ``par_unseq`` policies now enable vectorization also for Intel oneAPI DPC++/C++ Compiler.
+- Added support for passing zip iterators as segment value data in ``reduce_by_segment``, ``exclusive_scan_by_segment``,
+  and ``inclusive_scan_by_segment``.
+- Improved performance of ``merge``, ``sort``, ``stable_sort``, and ``sort_by_key`` algorithms on GPU devices.
+- Improved performance of the ``reduce``, ``min_element``, ``max_element``, ``minmax_element``,
+  ``is_partitioned``, and ``lexicographical_compare`` algorithms with DPC++ execution policies.
+
+Fixed Issues
+------------
+- Fixed the ``reduce_async`` function to not ignore the provided binary operation.
+
+Known Issues and Limitations
+----------------------------
+New in This Release
+^^^^^^^^^^^^^^^^^^^
+- When compiling with DPC++ compiler, libstdc++ version 8 or libc++, and with ``-fsycl-pstl-offload`` option,
+  parallel algorithms from std namespace called with ``oneapi::dpl::parallel_unsequenced_policy`` would be
```

Comment:
Is is certain that the algorithms are offloaded upon satisfying all the conditions above (DPC++ compiler, `-fsycl-pstl-offload` option and so on)? If so, let's replace "would" with "are", otherwise with "may".

Reply:
It will definitely happen on the mentioned systems, because they don't implement execution policies as part of the standard library. So oneDPL injects algorithms and policies into `std::` namespace. And strictly speaking, `std::execution::parallel_unsequenced_policy` and `oneapi::dpl::execution::parallel_unsequenced_policy` are the same types.
So calling `std::for_each` with `onedpl::par_unseq` is the same as calling `std::for_each` with `std::execution::par_unseq`.

So I would say that we can replace "would" with "are" in the wording.